test/gpu/native/gpuAddNums/gpuAddNums.chpl

```diff
@@ -4,27 +4,35 @@ extern {
 
   #define FATBIN_FILE "gpuAddNums_gpu_files/chpl__gpu.fatbin"
 
-  static void checkCudaErrors(CUresult err) {
-    assert(err == CUDA_SUCCESS);
+  static void checkCudaErrors(CUresult err, int i) {
+    if (err != CUDA_SUCCESS) {
+      printf("Error in call %d code:%d\n", i, err);
+      fflush(stdout);
```

Comment:
You may want to add a line like this here as well as in `chpl-gpu.c` just to save us the trouble of looking up error codes:
`printf("%s\n", cudaGetErrorString((int)err));`